conf/adzerk.py

```diff
@@ -30,6 +30,15 @@
                 "eventIds": [17, 20],
             }]
         }
+    },
+    # These are hardcoded and not expected to change.
+    # If they do end up changing, we should fetch them from https://api.adzerk.net/v1/priority.
+    "priority_id_to_weight": {
```

Comment:
It is my understanding that our AdZerk's priorities have never been updated. Hard-coding to save time for now works for me.